### PR TITLE
GPlates 2.2.0

### PR DESCRIPTION
**Pull request by peterdavidhamilton**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/gplates.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  url "https://downloads.sourceforge.net/gplates/#{version.major_minor}/GPlates-#{version}-Darwin-x86_64.dmg"
+  appcast 'https://sourceforge.net/projects/gplates/rss?path=/',
+          configuration version.major_minor
+  name 'GPlates'
+  homepage 'https://www.gplates.org/'
+
+  depends_on macos: '>= :mavericks'
+
+  app "GPlates-#{version}/GPlates.app"
+  binary "#{appdir}/GPlates.app/Contents/MacOS/gplates"
```
Comment (vitorgalvao) [inline Casks/gplates.rb]: For this to be accepted, it needs to be explicitly recommended by upstream. Is there a page you can point to?

Comment (peterdavidhamilton) [reply Casks/gplates.rb]: Hi Vitor. The latest [GPlates](http://gplates.org/) is compatible with 10.6 and above; so I set the dependency to Mavericks. 
<http://gplates.org/download.html>.

Comment (vitorgalvao) [reply Casks/gplates.rb]: I’m talking about this specific line. Linking the binary.

Unless upstream specifically recommends it, remove it.

File: Casks/gplates.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 'a4658d7dd7e123c53d847c7b98e91d3f9cf222beb0dadaea6a4868eddcf5d85a'
+
+  # downloads.sourceforge.net/gplates was verified as official when first introduced to the cask
+  url "https://downloads.sourceforge.net/gplates/#{version.major_minor}/GPlates-#{version}-Darwin-x86_64.dmg"
+  appcast 'https://sourceforge.net/projects/gplates/rss?path=/',
+          configuration version.major_minor
+  name 'GPlates'
+  homepage 'https://www.gplates.org/'
+
+  depends_on macos: '>= :mavericks'
```
Comment (vitorgalvao) [inline Casks/gplates.rb]: Remove this. Homebrew doesn’t work below this version, anyway.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/gplates.rb (hunk 1 above)
Comment (peterdavidhamilton) [reply Casks/gplates.rb]: I see. Older versions certainly had command-line only flags but I don't know if this latest version does anymore. I have removed it as suggested.

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Thank you, and welcome.

**Pull request merged**